docs/api/web-contents.md

```diff
@@ -1461,6 +1461,16 @@ Starts inspecting element at position (`x`, `y`).
 
 Opens the developer tools for the shared worker context.
 
+#### `contents.inspectSharedWorkerById(workerId)`
+
+* `workerId` String
+
+Inspects the shared worker based on its ID.
+
+#### `contents.getAllSharedWorkers()`
+
+Returns `WorkerInfo[]` - An array of WorkerInfo objects of sharedworkers.
```

Comment:
> "An array of WorkerInfo objects"

This just duplicates the type information (`WorkerInfo[]`), let's change the description to simply say
```
Returns `WorkerInfo[]` - Information about all Shared Workers. An empty array if there's no Shared Workers.
```